installVm-linux.sh

```diff
@@ -7,3 +7,16 @@ else
   echo "installing mulitpass"
   sudo snap install multipass
 fi
+
+if (multipass info mos-vm)
+then 
+  echo 'the VM already exists'
+else
+  echo 'creating mos-vm'
+  multipass launch --name mos-vm
+fi
+if (multipass info mos-vm)
```

Comment:
You wouldn't check like this here. The desired outcome would be "I should be logged into the vm" which is probably impossible to do from within the script. What you would do is call your `installVM-linux.sh` and then run the `multipass shell mos-vm` after the script. So something like this in the README.md would be good:

## Create a vm

`$SHELL installVm-linux.sh && multipass shell mos-vm`

Reply:
@MonetCarriere @DevinBeu